conversationgraph.py

```diff
@@ -47,10 +44,28 @@ def __init__(self, path: str):
                     )
                 )
 
-    def getByTag(self, tag: str) -> Corpus:
+    def get_by_tag(self, tag: str) -> Corpus:
         for x in self.corpuses:
             if x.tag == tag:
                 break
         else:
             x = None
         return x
+
+    def check_if_is_achieved(self, tag: str) -> bool:
```

Comment:
I think that name like `check_if_is_achievable()` will be better, because (if I undersdand correctly)s we are checking if given tag is allowed to be achieve but not that it has been already achieved